includes/ppc-user-manual.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * The Pre-Publish Checklist User Manual tab
+ *
+ * @since   1.0.0
+ * @package Pre-Publish Checklist.
+ * @author  Brainstorm Force.
+ */
+
+wp_enqueue_style( 'ppc_backend_css' );
+?>
+<!DOCTYPE html>
```

Comment:
The tabs are already inside `<html>`, This templates should not add the complete HTML markup, you can remove all the markup until `<form>`